src/lib/components/Imprint.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+
+import { GroupField, Input } from "react-invenio-forms";
+import { Grid } from "semantic-ui-react";
+
+import PropTypes from "prop-types";
+
+export class Imprint extends Component {
+  render() {
+    const {
+      fieldPath, // injected by the custom field loader via the `field` config property
+      title,
+      publisher,
+      place,
+      isbn,
+      pages,
+    } = this.props;
+    const fieldPathPrefix = `${fieldPath}`;
```

Comment:
minor: Are we modifying `fieldPath` anywhere? why do we need to create a new variable?

Reply:
We are not modifying it. 

It's a leftover from the docs, where we are indeed modifying it. So it can be removed I suppose.